nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -54,7 +54,6 @@
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
 ## New Modules {#sec-release-25.05-new-modules}
-
```

Comment:
Did you accidentally delete an extra line here?

Reply:
Oops. Seems that I deleted it by accident. Fixed in the latest commit.